From b5c5938579823a20ebda3631aee073c53f3c9ef8 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Thu, 25 Jul 2024 19:58:22 -0300
Subject: [PATCH] cam-server-base.service: create sockets with group rw

Create sockets with group rw in order to allow normal users to access
camera-server.

Upstream-Status: Pending

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 qmmf-server/service/cam-server-base.service | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/qmmf-server/service/cam-server-base.service b/qmmf-server/service/cam-server-base.service
index 9d4e962..177c80e 100644
--- a/qmmf-server/service/cam-server-base.service
+++ b/qmmf-server/service/cam-server-base.service
@@ -38,8 +38,9 @@ User=system
 Group=video
 Restart=always
 EnvironmentFile=/etc/cam-server-env
-ExecStartPre=/bin/mkdir -p /tmp/socket/cam_server
+ExecStartPre=/bin/mkdir -m 775 -p /tmp/socket/cam_server
 ExecStart=/usr/bin/cam-server
+UMask=0002
 
 [Install]
 WantedBy=multi-user.target
-- 
2.34.1

